docs/api/LightroomPartnerAPIsSpec.json

```diff
@@ -813,6 +813,101 @@
         }
       }
     },
+    "/v2/catalogs/{catalog_id}/assets/{asset_id}/xmp/develop": {
+      "put": {
+        "tags": [
+          "assets"
+        ],
+        "summary": "Create or Copy latest asset external xmp develop setting file",
+        "description": "Create or Copy latest asset external xmp develop setting file",
```

Comment:
Can you add a description as below. 
This API supports two workflows
1. Upload external XMP develop settings file for this asset. Request body content-type is application/rdf+xml for this case. 
2. Copy external XMP develop settings from another asset. Request body content-type is application/json  for this case